pkg/codegen/schema/schema.go

```diff
@@ -940,24 +953,26 @@ func bindDefaultValue(value interface{}, spec *DefaultSpec, typ Type) (*DefaultV
 	return dv, nil
 }
 
-func (t *types) bindProperties(properties map[string]PropertySpec, required []string) ([]*Property, error) {
+func (t *types) bindProperties(properties map[string]PropertySpec,
```

Comment:
Could you add a docstring? Hard to follow what the output values are.

Reply:
Done.